src/utils/__tests__/useOutsideClick-test.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import { mount, ReactWrapper } from 'enzyme';
+import useOutsideClick from '../useOutsideClick';
+
+describe('src/utils/useOutsideClick', () => {
+    let callback: () => void;
+
+    beforeEach(() => {
+        document.body.innerHTML = '<div id="container"></div>';
+        callback = jest.fn();
+    });
+
+    function TestComponent(): JSX.Element {
+        const ref = React.createRef<HTMLButtonElement>();
+
+        useOutsideClick([], ref, callback);
+
+        return (
+            <button ref={ref} id="test-button" type="button">
+                <span>Test</span>
```

Comment:
Should we test that clicking its inner content will also not trigger the callback?